# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libtar/hotfix.patch.cross
# Copyright (C) 2011 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- libtar-1.2.11-orig/compat/snprintf.c	2000-10-27 18:16:34.000000000 +0200
+++ libtar-1.2.11/compat/snprintf.c	2011-01-20 06:59:50.890448144 +0100
@@ -50,7 +50,7 @@

 /* Define this as a fall through, HAVE_STDARG_H is probably already set */

-#define HAVE_VARARGS_H
+#define HAVE_STDARG_H

 /* varargs declarations: */

